hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java

```diff
@@ -76,21 +78,28 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
     }
 
     @Override
-    public void handle(@NonNull final HandleContext context) throws HandleException {
-        requireNonNull(context);
+    public void handle(@NonNull final HandleContext handleContext) throws HandleException {
+        requireNonNull(handleContext);
 
-        final var op = context.body().fileAppendOrThrow();
-        final var target = op.fileID();
-        final var data = op.contents();
-        final var fileServiceConfig = context.configuration().getConfigData(FilesConfig.class);
+        final var fileAppend = handleContext.body().fileAppendOrThrow();
+        final var target = fileAppend.fileID();
+        final var data = fileAppend.contents();
+        final var fileServiceConfig = handleContext.configuration().getConfigData(FilesConfig.class);
         if (data == null || data.length() <= 0) {
             logger.debug("FileAppend: No data to append");
         }
 
         if (target == null) {
             throw new HandleException(INVALID_FILE_ID);
         }
-        final var fileStore = context.writableStore(WritableFileStoreImpl.class);
+
+        // the update file always will be for the node, not a particular ledger that's why we just compare th num
```

Comment:
Hmm it's not that important until we have multiple shards deployed, but I think we _do_ want to validate the entire `FileID` here, not only the number.

Reply:
We will talk about it during our daily call.

Reply:
Per modularization call on Tuesday, upgrade file should apply to the whole network. It should get clamped shard=0, realm=0